### PR TITLE
Update more-about-calling-line-ID-and-calling-party-name.md

### PR DESCRIPTION
**Pull request by jvijeyanandh**
If CLID is missing, Microsoft cannot address the same as it is terminating carrier issue. Hence updating document to clarify that ticket should be raised with terminating carrier and not with MS.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/more-about-calling-line-ID-and-calling-party-name.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ CallerID, as it is typically referred to, actually consists of two user-facing i
...
+CLID & CNAM transmission is the responsibility of the terminating carrier insofar as the terminating carrier must support CLID & CNAM functionality as well as provide up to date records for both values. Microsoft reliably provides CLID values when originating calls, but those values may not be kept intact once they pass through an intermediary carrier or the terminating carrier. Unfortunately, in the event the CLID value is changed, omitted or truncated by the intermediary or terminating carrier, Microsoft has little to no recourse in correcting such problems in the public telephone network.If there are issues related to unexpected CLID please raise a ticket with terminating carrier to address it.
```
Comment (JohanFreelancer9) [inline Teams/more-about-calling-line-ID-and-calling-party-name.md:33]:
```suggestion
CLID & CNAM transmission is the responsibility of the terminating carrier insofar as the terminating carrier must support CLID & CNAM functionality as well as provide up-to-date records for both values. Microsoft reliably provides CLID values when originating calls, but those values may not be kept intact once they pass through an intermediary carrier or the terminating carrier. Unfortunately, in the event the CLID value is changed, omitted, or truncated by the intermediary or terminating carrier, Microsoft has little to no recourse in correcting such problems in the public telephone network. If there are issues related to unexpected CLID, please raise a ticket with the terminating carrier to address the issue. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @jvijeyanandh, please commit my suggested changes. Many thanks.

**Review by scanum — APPROVED** [review]

**Comment by tiburd** [comment]
Closing this PR due to age, inactivity, and file conflicts.

cc: @scanum

**Pull request closed without merging**